lib/languages/gatsby/helpers/index.js

```diff
@@ -0,0 +1,257 @@
+const path = require('path');
+const util = require('util');
+const qs = require('qs');
+const commonHelpers = require('../../../common-helpers');
+
+const helpers = Object.assign({}, commonHelpers);
```

Comment:
which of these did you modify from the mkdocs ones? cause you could just pull in the mkdocs helpers instead of the common ones, and then override